gulpfile.js

```diff
@@ -62,8 +64,27 @@ gulp.task('sass', function () {
 
 gulp.task('generate-assets', ['assets', 'templates', 'sync', 'sass'])
 
-gulp.task('generate-assets-and-start', ['generate-assets'], function () {
-  spawn('node', ['app/bin/www'], { stdio: 'inherit' })
+gulp.task('generate-assets-and-start', ['generate-assets', 'server'], function () {
+
+})
+
+gulp.task('nsp', function (cb) {
+  gulpNSP({package: __dirname + '/package.json'}, cb);
+})
+
+gulp.task('watch', function() {
+  gulp.watch(['app/app.js', 'app/services/**/*'], ['generate-assets-and-start']);
```

Comment:
😍 can we also track `app/assets/**/*` to pick up sass and frontend Javascript changes?